### PR TITLE
[fix] typo for time_limits

### PR DESCRIPTION
**Pull request by chongruo**
fix typo for time_limits

### CODE REVIEW

File: docs/tutorials/object_detection/beginner.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ In this section, we demonstrate how to apply AutoGluon to fit our detection mode
 We `fit` a classifier using AutoGluon as follows. In each experiment (one trial in our searching space), we train the model for 30 epoches. 
 
 ```{.python .input}
-time_limits = 5*60*60 # 5 days
+time_limits = 5*60*60   # 5 mins
```
Comment (Innixma) [inline docs/tutorials/object_detection/beginner.md]: This is 5 hours, not days

File: examples/object_detection/demo.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 data_root = os.path.join(root, filename)
 dataset_train = task.Dataset(data_root, classes=('motorbike',))
 
-time_limits = 5*60*60 # 5 days
+time_limits = 5*60*60 # 5 mins
```
Comment (Innixma) [inline examples/object_detection/demo.py]: This is 5 hours, not minutes

File: docs/tutorials/object_detection/beginner.md (hunk 1 above)
Comment (chongruo) [reply docs/tutorials/object_detection/beginner.md]: oops...has fixed. Thanks!

File: examples/object_detection/demo.py (hunk 2 above)
Comment (chongruo) [reply examples/object_detection/demo.py]: thx!

**Comment by szha** [comment]
Job PR-193-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-193/1/index.html

**Comment by szha** [comment]
Job PR-193-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-193/2/index.html

**Comment by szha** [comment]
Job PR-193-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-193/3/index.html

**Pull request merged**